content/donate/funded_development.md

```diff
@@ -0,0 +1,153 @@
+# Development Contributions financed by Haiku, Inc.
```

Comment:
This page is missing at least the following:

2019: Preetpal's work on Input preferences as part of Outreachy

2014-2016 (don't remember the exact dates):
- My work in 2014 on WebKit and general OS mprovements
- Pawel's work on the affine scheduler
- Barrett's work on media kit

Forgot the year:
- Waddlesplash's contract for the haikuports build infrastructure (it wasn't put in production, but as far as I know it was still funded)

2008 and 2009: "Haiku Code Drive", funding extra students aside GSoC when Google did not allow us as much students as we wanted to accept.

I can submit a later pull request adding these, when this one is merged.

Reply:
@pulkomandy I think you're right, it's better to merge this first and then either you or I can get to finding all this other information for Contracts etc. 

Should we have a page describing GSOC?

Reply:
The GSoC info is already at haiku-os.org/community/gsoc, there is no need to replicate it here. Maybe we can add a link to it, but it's a bit outside the scope here (since haiku inc didn't technically fund it, the payment was directly from Google to the students).

Reply:
> Waddlesplash's contract for the haikuports build infrastructure (it wasn't put in production, but as far as I know it was still funded)

The contract was partially for the build infrastructure and partially just for "cleaning up HaikuPorts." While the Kitchen app never did get put into production, indeed, the haikuports cleanup obviously was merged (and part of that was migrating from Bitbucket to GitHub.)